### DISCUSSION TITLE
macOS version conditional in Brewfile

### DISCUSSION THREAD

**Discussion by rgoldberg** [Homebrew/discussions#5675] [Everyday usage] [answered]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

How can I performa a macOS version conditional in a `Brewfile`?

Something like:

```ruby
on_ventura :or_newer do
    # some Brewfile commands
else
    # some other Brewfile commands
end
```

**Comment by p-linnane** [comment]
This is not supported.

**Reply by rgoldberg** [reply]
Just to check: you're saying that there is no syntax to conditionally run `Brewfile` lines based on the OS version, not just that my fake example syntax is incorrect, right?

**Reply by ZhongRuoyu** [reply, marked as answer]
You should be able to do:

```ruby
if OS.mac?
  if MacOS.version >= :ventura
    brew "some-formula"
  else
    brew "some-other-formula"
  end
end
```
